### PR TITLE
Feature/cdhk 70

### PR DESCRIPTION
**Pull request by businessglitch**
OCR feature for CarletonU calendar

### CODE REVIEW

File: python-server/helper.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import cv2
+import json
+
+# Saves an array as a JSON in a file
+
+
```
Comment (BenJeau) [inline python-server/helper.py]: Maybe remove the two line before the method to put the comments closer to the method

File: python-server/helper.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+def save_json(data, filename):
+    with open(filename + '.json', 'w') as outfile:
+        json.dump(data, outfile)
+
+# Returns an image read by cv2
+
+
```
Comment (BenJeau) [inline python-server/helper.py]: Same as above

File: python-server/helper.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+def read_image(filename):
+    return cv2.imread(filename)
+
+# Extract contours and hierarchy from an image
+
+
```
Comment (BenJeau) [inline python-server/helper.py]: Same as above

File: python-server/helper.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+def extract_contours(img):
+    # Gray scale the image
+    gray = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+
+    #Apply threshold to the image
```
Comment (BenJeau) [inline python-server/helper.py]: Add space after #

File: python-server/helper.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    ret, mask = cv2.threshold(gray, 220, 225, cv2.THRESH_BINARY_INV)
+
+    #IMAGE CONTOURS
```
Comment (BenJeau) [inline python-server/helper.py]: Same as above and make the comment not all caps

File: python-server/helper.py (hunk 1 above)
Comment (businessglitch) [reply python-server/helper.py]: Thanks! Fixed them in the new commit

**Pull request merged**